project/ProjectPlugin.scala

```diff
@@ -21,31 +21,38 @@ object ProjectPlugin extends AutoPlugin {
       Map(
         'evaluator     -> "0.5.0-SNAPSHOT",
         'stdlib        -> "0.5.0-SNAPSHOT",
-        'cats          -> "0.4.2-SNAPSHOT",
-        'shapeless     -> "0.4.2-SNAPSHOT",
-        'doobie        -> "0.4.2-SNAPSHOT",
-        'scalacheck    -> "0.4.2-SNAPSHOT",
-        'scalatutorial -> "0.4.2-SNAPSHOT",
-        'fpinscala     -> "0.4.2-SNAPSHOT",
-        'circe         -> "0.4.2-SNAPSHOT",
-        'fetch         -> "0.4.2-SNAPSHOT",
-        'monocle       -> "0.4.2-SNAPSHOT"
+        'cats          -> "0.5.0-SNAPSHOT",
+        'shapeless     -> "0.5.0-SNAPSHOT",
+        'doobie        -> "0.5.0-SNAPSHOT",
+        'scalacheck    -> "0.5.0-SNAPSHOT",
+        'scalatutorial -> "0.5.0-SNAPSHOT",
+        'fpinscala     -> "0.5.0-SNAPSHOT",
+        'circe         -> "0.5.0-SNAPSHOT",
+        'fetch         -> "0.5.0-SNAPSHOT",
+        'monocle       -> "0.5.0-SNAPSHOT"
       ) ++ Map(
         // JVM Versions
-        'bootstrap      -> "3.3.7",
-        'classutil      -> "1.1.2",
-        'commonsio      -> "2.6",
-        'freestyle      -> "0.8.2",
-        'highlightjs    -> "9.2.0",
-        'knockoff       -> "0.8.12",
-        'newrelic       -> "5.6.0",
-        'postgres       -> "42.2.8",
-        'scalajsscripts -> "1.1.4",
-        'scalariform    -> "1.8.3",
-        'upickle        -> "0.7.5",
-        'webjars        -> "2.7.3",
-        'scalamacros    -> "2.1.1",
-        'monix          -> "3.0.0"
+        'catsversion         -> "2.0.0",
+        'doobieversion       -> "0.8.6",
+        'bootstrap           -> "3.3.7",
+        'github4s            -> "0.20.1",
+        'classutil           -> "1.5.1",
+        'commonsio           -> "2.6",
+        'highlightjs         -> "9.15.10",
+        'knockoff            -> "0.8.12",
+        'newrelic            -> "5.8.0",
+        'postgres            -> "42.2.8",
+        'scalajsscripts      -> "1.1.4",
+        'scalariform         -> "1.8.3",
+        'scalatest           -> "3.0.8",
```

Comment:
We should bump to 3.1.0. Keep in mind that the artifact has been deduplicated into two, where you might need `org.scalatestplus.scalatestplus-scalacheck`

> In 3.0.8 we moved Checkers under org.scalatest.prop to org.scalatestplus.scalacheck package, and deprecated Checkers under org.scalatest.prop. Checkers was still released in the ScalaTest artifact, with the org.scalatestplus.scalacheck package name. In 3.1.0, we are removing the deprecated name and the actual implementation in the new package from the ScalaTest artifact. From now on these will be released as a separate, optional module to allow ScalaTest and its ScalaCheck integration library to evolve more independently.

Reply:
Should I update exercises too? The libraries are using 3.0.8

Reply:
Yes, we can do it later on